src/macaron/slsa_analyzer/git_service/gitlab.py

```diff
@@ -117,7 +123,74 @@ def clone_repo(self, clone_dir: str, url: str) -> None:
             If there is an error cloning the repository.
         """
         clone_url = self.construct_clone_url(url)
-        git_url.clone_remote_repo(clone_dir, clone_url)
+        repo = git_url.clone_remote_repo(clone_dir, clone_url)
+
+        # If ``git_url.clone_remote_repo`` return an Repo instance, this means that the repository is freshly cloned
```

Comment:
```suggestion
       # If ``git_url.clone_remote_repo`` returns a Repo instance,
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/354/commits/6678fe22d14923dfb62282973f68e093b5abd38b